package.json

```diff
@@ -92,9 +101,10 @@
     "jest": "^24.9.0",
     "jest-emotion": "10.0.10",
     "jest-junit": "^10.0.0",
-    "lint-staged": "^7.3.0",
+    "lint-staged": "^9.4.3",
     "lodash.round": "^4.0.4",
     "prettier": "1.14.3",
+    "react-docgen-typescript-loader": "^3.3.0",
```

Comment:
This has since been removed so let's uninstall it.

Reply:
done